fractal_server/app/routes/api/v2/task.py

```diff
@@ -150,8 +150,8 @@ async def create_task(
     # Set task.owner attribute
     if user.username:
         owner = user.username
-    elif user.slurm_user:
-        owner = user.slurm_user
+    elif user.settings.slurm_user:
```

Comment:
make sure you include `verify_user_has_settings`, before calling `user.settings`